internal/repositories/hash_storage.go

```diff
@@ -0,0 +1,76 @@
+package repositories
+
+import (
+	"database/sql"
+	"github.com/ScienceSoft-Inc/integrity-sum/internal/models"
+	"github.com/ScienceSoft-Inc/integrity-sum/pkg/api"
+	"github.com/sirupsen/logrus"
+)
+
+type HashStorage struct {
+	db     *sql.DB
+	alg    string
+	logger *logrus.Logger
+}
+
+// NewHashStorage creates a new struct HashService
+func NewHashStorage(db *sql.DB, alg string, logger *logrus.Logger) *HashStorage {
+	return &HashStorage{
+		db:     db,
+		alg:    alg,
+		logger: logger,
+	}
+}
+
+// Create accesses the repository to save data to the database
+func (hs HashStorage) Create(allHashData []*api.HashData, deploymentData *models.DeploymentData) error {
```

Comment:
Looks strange: writes api.HashData. - Why `api`? Where is the own DB layer model?

Reply:
resolved: removed pkg/app - as functions included in the package were not used https://github.com/ScienceSoft-Inc/integrity-sum/commit/1c2acf8587eec4112f90c497da37ae13e28d7f07